integrations/weaviate/src/haystack_integrations/document_stores/weaviate/document_store.py

```diff
@@ -388,7 +387,8 @@ def filter_documents(self, filters: Optional[Dict[str, Any]] = None) -> List[Doc
         :returns: A list of Documents that match the given filters.
         """
         if filters and "operator" not in filters and "conditions" not in filters:
-            filters = convert(filters)
+            msg = "Legacy filters support has been removed. Please see documentation for new filter syntax."
+            raise ValueError(msg)
```

Comment:
Can we mention if we are talking about weaviate documentation or haystack docs, as this sounds a bit ambiguous?

Reply:
@yes, included a link to docs now, see the diff @Amnah199